src/ceph-volume/ceph_volume/devices/lvm/batch.py

```diff
@@ -260,18 +257,6 @@ def __init__(self, argv):
         for dev_list in ['', 'db_', 'wal_', 'journal_']:
             setattr(self, '{}usable'.format(dev_list), [])
 
-    def get_devices(self):
-        # remove devices with partitions
-        devices = [(device, details) for device, details in
-                       disk.get_devices().items() if details.get('partitions') == {}]
```

Comment:
you have to remove the import L6 to `ceph_volume.util.disk` since it's now unused